pkg/lib/machine-change-auth.html

```diff
@@ -10,7 +10,11 @@ <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
     {{/available}}
 
     {{#available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}. You can change your authentication credentials below. {{#can_sync}}You may prefer to {{#sync_link}}synchronize accounts and passwords{{/sync_link}}.{{/can_sync}}</p>
+    <p translate="yes">Cockpit was unable to log into
+    {{#strong}}{{host}}{{/strong}} automatically. You need to provide
+    your authentication credentials every time you connect to this
+    machine. You may want to set up your SSH keys for automatic
+    login.</p>
```

Comment:
Need to connect this stronger language-wise, or location-wise to the checkbox below probably.
The checkbox below does not mention anything about _adding_ SSH keys.

Reply:
What about something like this?
`Unable to log in to 192.168.1.15 using SSH key authentication.`

Reply:
Done.